client/web/web.go

```diff
@@ -65,13 +65,32 @@ type Server struct {
 	browserSessions sync.Map
 }
 
+// ServerType specifies the type of a web.Server.
+// If "tailscaled", a full management client is being served.
+// If "system-app", a readonly login client is being served.
+type ServerType string
+
+const (
+	Tailscaled ServerType = "tailscaled"
```

Comment:
With whatever we decide above, we'll probably want to be more explicit in these names.  An exported var just named `Tailscaled` is a little confusing.  So `ServerTypeTailscaled` or `ManageServerMode` or whatever ?